M2/Macaulay2/tests/normal/doc-links.m2

```diff
@@ -6,6 +6,7 @@ assert ( match (
 	  "<span><a title=\"an example Macaulay2 package\" href=\".*/FirstPackage/html/index.html\">FirstPackage</a> -- an example Macaulay2 package</span>",
 	  html TOH "FirstPackage::FirstPackage"))
 needsPackage "FirstPackage"
+if getenv "MakeDocumentation" == "false" then end
```

Comment:
When would this be necessary? I understand that if `IARGS` explicitly includes `MakeDocumentation => false`, this wouldn't work, but there are plenty of other ways to stop this from working.

What's the purpose of `IARGS` in autotools for that matter? Is it being used anywhere, or does anyone use it? If not, simply removing it is a much easier fix and reduces complexity instead of adding more.

Reply:
`IARGS` is for adding user-defined arguments to `installPackage` during the build.  I'm using it right now to add support for building a Debian package without documentation.  (This is supposed to be supported if a user passes the environment variable `DEB_BUILD_OPTIONS` that includes the string `nodoc`.)

Reply:
Makes sense. With the CMake build, if you don't want that, you just stop after `ninja M2-core`. No need to run `ninja install-packages`.

Just a warning though, without installing the documentation, M2 is significantly slower. The executable itself takes 3 to 4 times longer to start. Loading some packages takes many seconds (particularly if they have Polymake somewhere down the chain of dependencies).

Reply:
Yeah, I don't really envision ever actually using such a package in production.  I see two big uses:

* Quick smoke-test builds.  (There's also a `nocheck` option.  Passing both of these reduces the total build time from hours to minutes!)
* In the future, if/when we switch to an architecture-independent database, we can skip building the documentation on non-amd64 architectures completely and just install the documentation that was built on amd64.